src/utils/crypto.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { mockCrypto } from '../../jest.setup';
+import {
+  mockPbkdf2Key,
+  mockEncryptedString,
+  mockNativeAes,
+  mockRandomKey,
+  mockUnencryptedString
+} from '../mocks/native-modules.mock';
+import { decryptString$, encryptString$, generateRandomValues } from './crypto.util';
+import { rxJsTestingHelper } from './testing.utis';
+
+describe('generateRandomValues', () => {
+  beforeEach(() => {
+    mockCrypto.getRandomValues.mockClear();
+  });
+
+  it('should generate 16 cryptographically safe random bytes by default', () => {
+    const output = generateRandomValues();
+    expect(output).toHaveLength(16);
+    expect(mockCrypto.getRandomValues).toBeCalledWith(output);
+  });
+
+  it('should generate another amount of random bytes if specified', () => {
+    const expectedAmount = 12;
+    const output = generateRandomValues(expectedAmount);
+    expect(output).toHaveLength(expectedAmount);
+    expect(mockCrypto.getRandomValues).toBeCalledWith(output);
+  });
+});
+
+beforeEach(() => {
+  mockNativeAes.decrypt.mockClear();
+  mockNativeAes.encrypt.mockClear();
+  mockNativeAes.pbkdf2.mockClear();
+});
```

Comment:
does this hook work outside the `describe` block?  
does it work in space of the file?

Reply:
Definitely, it works
![Screenshot_2021-08-11_11-49-48](https://user-images.githubusercontent.com/10885399/128999430-cb5bcb8a-adfe-4530-ad1c-0fa6c67d2b06.png)

Reply:
In addition, it will hardly impact tests from other files
![Screenshot_2021-08-11_11-56-34](https://user-images.githubusercontent.com/10885399/129000481-30ad0bed-e31f-4028-be9d-530b044293e5.png)